Classes/Authentication/OpenIdConnectProvider.php

```diff
@@ -218,7 +228,14 @@ private function getConfiguredRoles(IdentityToken $identityToken): array
                     continue;
                 }
 
-                foreach ($identityToken->values[$claim] as $i => $roleIdentifier) {
+                foreach ($identityToken->values[$claim] as $roleIdentifier) {
+                    if ($mapping !== null) {
```

Comment:
Dann kannst Du auch oben einen isset check auf $claim['mapping'] machen, dann ist das hier schon mal nicht mehr nötig. Effektiv müsstest Du aber sowieso auch oben noch einen !is_array() check darauf machen weil alles andere Dir hier auch nicht hilft oder?

Reply:
You mean an `isset($claim['mapping'])` instead of `!array_key_exists('mapping', $claim)` in line https://github.com/flownative/flow-openidconnect-client/blob/a9ae8bfcce460c538d65d657a2cb1f9174c2823c/Classes/Authentication/OpenIdConnectProvider.php#L213 ?

Reply:
feel free to insert a suggestion if that's not what you meant

Reply:
FYI: I added an additional `is_array` check with #26